### PR TITLE
Create register page design

### PR DESCRIPTION
**Pull request by piya43044**
Here's the pull request Description format:
- https://internship-splitwise.atlassian.net/browse/IS-45

**Checklist**
- [x] Code adheres to **[Git and Pull Request Guidelines]**
- [x] Code adheres to **[Angular Guidelines]**
- [x] Code adheres to **[Front-end Checklist]**

### CODE REVIEW

File: src/app/user-sign-up/register/register.component.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/* Import common styling */
+@import "../../layout/variables";
+
+/* Register Form container */
+.form-container {
```
Comment (AshaPatel216) [inline src/app/user-sign-up/register/register.component.scss:6]: The button style will be common to entire project. So whatever common style is there for button, please shift it to global style.

File: src/app/user-sign-up/register/register.component.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    border-radius: 15px;
+    box-shadow: 0 5px 15px $off-black;
+    padding: 20px;
+    width: 400px;
+
+    /* Form group contains input, label tags, validation */
+    .form-group {
+        display: flex;
```
Comment (AshaPatel216) [inline src/app/user-sign-up/register/register.component.scss]: Use bootstrap class for flex properties.

File: src/app/user-sign-up/register/register.component.scss (hunk 1 above)
Comment (AshaPatel216) [reply src/app/user-sign-up/register/register.component.scss:6]: IS this change not done? And if no need to shift then please mention reason in the reply here

Comment (piya43044) [reply src/app/user-sign-up/register/register.component.scss:6]: I have created a common button style and tried to import it on the global style but it is not working. For that reason, I have imported them separately.

Comment (AshaPatel216) [reply src/app/user-sign-up/register/register.component.scss:6]: You just need to shift that style to styles.scss file. it should work

**Review by AshaPatel216 — APPROVED** [review]

**Pull request merged**